src/app/login/page.tsx

```diff
@@ -0,0 +1,72 @@
+import Form from 'next/form';
+import Link from 'next/link';
+
+// interface metadata {
+//     title: string;
+// }
+export const metadata = {
+    title: 'Log In',
+};
```

Comment:
make sure to remove any unused code for the PR (this can be done in this PR or if you can follow up with this in the next PR that would be great!)